packages/launchdarkly-adapter/modules/adapter/adapter.spec.js

```diff
@@ -13,7 +13,7 @@ const userWithoutKey = {
 };
 const flags = { 'some-flag-1': true, 'some-flag-2': false };
 const createClient = jest.fn(apiOverwrites => ({
-  waitUntilReady: jest.fn(() => Promise.resolve()),
+  waitForInitialization: jest.fn(() => Promise.resolve()),
```

Comment:
...cause the adapter now uses the fn while defauling into the same behaviour of not rethrowing to have this non-breaking.